sos/report/plugins/process.py

```diff
@@ -88,4 +89,28 @@ def setup(self):
             "pidstat -p ALL -rudvwsRU --human -h",
             "pidstat -tl"
         ])
+
+    def collect(self):
+        with self.collection_file('pids_to_packages.json') as pfile:
+            if not self.policy.package_manager.query_path_command:
+                pfile.write('Package manager not configured for path queries')
+                return
+            _ps = self.exec_cmd('ps --no-headers aex')
+            pidpkg = {}
+            paths = {}
+            if not _ps['status'] == 0:
+                pfile.write("Unable to get process list: %s" % _ps['output'])
+                return
+            for proc in _ps['output'].splitlines():
+                proc = proc.strip().split()
+                pid = proc[0]
+                path = proc[4]
+                if not self.path_exists(path):
+                    continue
+                if path not in paths:
+                    paths[path] = self.policy.package_manager.pkg_by_path(path)
```

Comment:
Rename `paths` variable to `pkgs`? As it tells us (for a given path) a package.

Reply:
`paths` is named that way, as we're holding the path string from the process listing there, which is what we're using to check the package. In other words "this (paths) dict has keys that are paths, and those keys' values are package(s) that own that path".